src/odemis/util/transform.py

```diff
@@ -919,7 +936,7 @@ def __call__(self, x):
         v = numpy.dot(M, self.coeffs)
 
         if x.ndim == 1:
-            return np.array((v.real, v.imag))
+            return numpy.array((v.real, v.imag))
```

Comment:
Good catch! I guess this isn't tested anywhere.

Reply:
Yeah... Pydev found it (so it's a little bit clever).
That said, if you want to add a test case, that'd be welcome ;-)